oberon07/Time.c

```diff
@@ -4,11 +4,11 @@
 
 #include <stdio.h>
 
-const int Time__CTimeStruct_id;
-const int *const Time__CTimeStruct_ids[1] = {&Time__CTimeStruct_id};
-const OBNC_Td Time__CTimeStruct_td = {Time__CTimeStruct_ids, 1};
+const int Time_CTimeStruct_id;
```

Comment:
OBNC 0.14.0 expects the symbols with two underscores. Do you use newer or older version of the compiler?

Reply:
I thought I was using 0.14.0, and I thought that since it worked on my machine with just one underscore you must have been using an older compiler. (I didn't think to check the main page.) After you asked, I recalled that I had been using a patched 0.13.x (patched after reporting a bug to maintainer). Since I was patched, I didn't think to update when he released 0.14.0. He's actually at 0.14.1 now, and after downloading & installing I can confirm that it requires two underscores. For what it's worth, it needs two underscores in several places:

```
const int Time__CTimeStruct_id;
const int *const Time__CTimeStruct_ids[1] = {&Time__CTimeStruct_id};
const OBNC_Td Time__CTimeStruct_td = {Time__CTimeStruct_ids, 1};

void Time__Time_(Time__CTimeStruct_ *time_, const OBNC_Td *time_td)
...
void Time__Init(void)
```
I'm sorry about these problems.